### DISCUSSION TITLE
[FEAT]  Can I run fastfetch in a docker container?

### DISCUSSION THREAD

**Discussion by snowdream** [fastfetch-cli/fastfetch#1351] [Q&A]
Be sure to read [FAQ](https://github.com/fastfetch-cli/fastfetch?tab=readme-ov-file#faq) before submitting a new issue.

## Wanted features:

Can I run fastfetch in a docker container?

**Comment by CarterLi** [comment]
You can try it. If it doesn't work, you can't.

**Comment by snowdream** [comment]
![image](https://github.com/user-attachments/assets/7cedde94-3ad6-44ed-a5b2-89bccaeed603)
<img width="586" alt="image" src="https://github.com/user-attachments/assets/6e705932-9c03-401c-96c7-ec01bb9b85dc">
<img width="586" alt="image" src="https://github.com/user-attachments/assets/2b4b133f-e41f-4c22-833f-52d99d4eabcd">

Yes, I can run fastfetch in a docker container.
But the informations are not exactly about mbp, but about which resources docker can get.

**Reply by CarterLi** [reply]
If you dont want the information of your docker, why do you want to run fastfetch in the docker?

**Reply by snowdream** [reply]
Develop faster. Run anywhere.
![image](https://github.com/user-attachments/assets/2c6a41fa-071b-445c-86fd-9deb1ecbd3d2)
